lib/jenkins_pipeline/pipeline_builder.rb

```diff
@@ -17,7 +16,13 @@ def build(pipeline, jenkins_jobs)
     private
 
     def find_job(name, jobs)
-      jobs.find { |job| job["name"] == name }
+      jobs.find {|job| job["name"] == name }
+    end
+
+    def upstream_status(jobs, job_config, matching_job)
```

Comment:
Unused method argument - `matching_job`. If it's necessary, use `_` or `_matching_job` as an argument name to indicate that it won't be used.